src/wizard/javascript/profiling-onboarding/react/2.configure-document-policy.md

```diff
@@ -0,0 +1,19 @@
+---
+name: React (JS Self Profiling)
+doc_link: https://docs.sentry.io/platforms/javascript/guides/react/profiling/
+support_level: beta
+type: framework
+---
+
+#### Add Document-Policy:js-profiling header
+
+In order to enable the profiler to start, the document response header has to include a header key `Document-Policy` with value of `js-profiling`.
+
+Depending on your server, how you do this will vary. If you are using a server like express, you can do this like via the [response.set](https://expressjs.com/en/4x/api.html#res.set) function.
```

Comment:
```suggestion
How you do this will depend on your server. If you're using a server like express, you'll be able to use the [response.set](https://expressjs.com/en/4x/api.html#res.set) function.
```